pkdiagram/scene/event.py

```
@@ -75,166 +118,136 @@ def read(self, chunk, byId):
             return True
         else:
             return False
 
-    def _do_setParent(self, parent):
-        was = self.parent
-        self.parent = parent
-        if was and not was.isEmotion and not was.isScene:
-            was._onRemoveEvent(self)
-        if parent and not parent.isEmotion and not parent.isScene:
-            parent._onAddEvent(self)
+    def _do_setPerson(self, person: "Person"):
```

Ruff (0.13.3)

Undefined name `Person`

(F821)